src/locale/languages.ts

```diff
@@ -25,6 +26,7 @@ export const APP_LANGUAGES: AppLanguageConfig[] = [
   {code2: AppLanguage.hi, name: 'हिंदी'},
   {code2: AppLanguage.ja, name: '日本語'},
   {code2: AppLanguage.fr, name: 'Français'},
+  {code2: AppLanguage.pt, name: 'Portuguese'},
```

Comment:
I am actually following other options, because the language is identified as pt and not pt-BR.
In this case I believe it is better to keep it for now until developers add pt-BR

![image](https://github.com/bluesky-social/social-app/assets/11423362/2ccbefa4-4a21-4281-913b-371525a2b597)

Thanks

Reply:
> I am actually following other options, because the language is identified as pt and not pt-BR. In this case I believe it is better to keep it for now until developers add pt-BR
> 
> ![image](https://private-user-images.githubusercontent.com/11423362/293712997-2ccbefa4-4a21-4281-913b-371525a2b597.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQyMDYwNzEsIm5iZiI6MTcwNDIwNTc3MSwicGF0aCI6Ii8xMTQyMzM2Mi8yOTM3MTI5OTctMmNjYmVmYTQtNGEyMS00MjgxLTkxM2ItMzcxNTI1YTJiNTk3LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTAyVDE0MjkzMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU5ZDFkNWZkMTY2MmIyYWVlNjZiNGNjYTk1OTEzMmQyMmU1MmZhZTJjMGU5MTIxZjg3NzI5ZDhiZWM0Njc1NzMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.tzdm07tjc3FJ7-3TD0z0zE5kEi5WY-S-6f_4a3Zj3WE)
> 
> Thanks

It would make sense to change the name to `Português`.

Even though the other fields for content languages are set with the language name in English, the selector for App Language has the name of the language in their respective language.

**E.g French is Français
Portuguese should be Português.**

**It's something though that someone from the bsky team should make it clear to be a default for all translations.** But putting it into thought would make a lot of sense. Not everyone knows the name of their language in English (especially if the language was changed by mistake).

Reply:
Yes, I agree that Português is better, but I believe the other options may still appear in Portuguese and this could cause confusion with the same options with different names. If this doesn't break the other options it's welcome, as it's the smallest change.
Can you confirm in the code for me if by changing this translation the other options followed the same translation?

Reply:
They won't atm. They come from another source. This is something that should be localized in the future.

https://github.com/bluesky-social/social-app/blob/4e9e92f976b5c26879e9126f88aaeea0c8b8c3bc/src/locale/languages.ts#L427